editions/dev/tiddlers/new/InputTypeModules.tid

```diff
@@ -0,0 +1,47 @@
+created: 202112111013000000
+modified: 202112111013000000
+title: inputtype-modules
+type: text/vnd.tiddlywiki
+
+Modules of the type `inputtype` provide a way to convert the TiddlyWiki internal representation of a value into a value used within the HTML-value-attribute of an input tag and back.
+
+These modules are used to allow the date and time types of HTML5 input tags to function seamlessly with TiddlyWiki's date/time format.
+
+A module of type `inputtype` exports the `type` property that specifies on which type of `input` element they are applied to. In addition to that, they contain two exported functions `toValue` and `fromValue`. These functions are called each time a value needs to be converted from TW5`s representation into the HTML-value-attribute and vice versa.
```

Comment:
There is a typo in ``converted from TW5`s representation`` See the backtick in ``TW`s`` It should be `TW's`